src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java

```diff
@@ -76,9 +75,21 @@ public String buildCollectorFile(ConcurStandardAccountingExtractFile saeFileCont
             LOG.error("buildCollectorFile(): There was a problem preparing the data for the Collector file;"
                     + " will not create a file. See earlier logs for details.");
             return StringUtils.EMPTY;
+        } else if (collectorDataRowsDoNotExistToOutput(collectorBatch)) {
+            LOG.info("buildCollectorFile(): No SAE data rows found to process as collector input data. Collector .data and .done files NOT being created.");
+            return StringUtils.EMPTY;

```

Comment:
Do you need to change any of the upstream processing in ConcurStandardAccountingExtractToPdpAndCollectorStep as well, so that the PDP file (if any) can still get its .done file created? Or are you not expecting the PDP file to be created if things are in a state where the Collector data would be empty?